### ISSUE TITLE
Docs "Format Configuration" has a mistake

### ISSUE THREAD

**Issue by jorenbroekema** [amzn/style-dictionary#785]
Hey, I believe I spotted an error in the docs here https://amzn.github.io/style-dictionary/#/formats?id=format-configuration

```json
{
  "files": [{
    "destination": "map.scss",
    "format": "scss/map-deep",
    "mapName": "my-tokens"
  }]
}
```

should be

```json
{
  "files": [{
    "destination": "map.scss",
    "format": "scss/map-deep",
    "options": {
      "mapName": "my-tokens"
    }
  }]
}
```

At least, this is what works for me for css/variables selector option: [Playground link](https://www.style-dictionary-play.dev/#project=rVVNj6MwDP0rFXOrEHQvq925rjR73ftSjQKENhQSZIeqVcV/n3wMNARKkWYilYJjv9jPD3MLMgJ5rC+RFCfKMSpR8OA1uCV8s0nMdhK8bsyjNqQCcgqOSRuB5KzFkVGba5GyinpmvXEmVWvsSXDTsSzigtMuCe5uXegi5RRPUjSroLAeA/W3nzc9cBIcqfLnB6+WTFQCJqW4xxiPqBBcRilBN+3JEZJe5FfwJQXJCFxnztB/6tIFoYWMdS4PmugdqbqofL28DkCuk7QqdjjKZdZf/pj1uHc1VeKon4D8NmtBAAROTyB+mDXf+tDRKs2X2H95e9up5fAduhxRyhejdzsdv6Jbtr3ruqV9vW5NGjgvIqMJVXE3Xw/STHBF7XUtlCHgAViv1fVY5Brpvj4TNy/YweMIRQuZwfzfk7TdxluXUAu6D21AUxFZCKhxRGyG6PEqgXDUjn9BtI1N2ng5hTZAC3axe5iPttKWVfk/Io921zzGKj4eeRVqKqKTvF6esNXAk4wTyQS3UO9nAoykKjAa59NLBGoih3zjwXviKRoNaus2EqhoJq3g1O8oUKHc3xznJdr7w61cQ16Jzwkqv4GfOz3lMjslORPMgDUypvhzdlzspyo0n5aFEWv2R9rSXzSPH+dFGEbEnU6sH7v/ai5DwJBV9wE=)

**Comment by dbanksdesign** [comment]
Hey @jorenbroekema  thanks for bringing this up! I believe it is actually correct as is. Looking at the source code for the `scss/map-deep` format, it is accessing `mapName` from the file object directly and not the `options` object on the file: https://github.com/amzn/style-dictionary/blob/main/lib/common/templates/scss/map-deep.template#L20

We have been working towards moving configuration onto the `options` object with more recent additions though so this might be something we want to update in the future. 

Also, the playground link you sent doesn't appear to show the issue. Here is what I am seeing:

<img width="1245" alt="CleanShot 2022-03-08 at 14 11 30@2x" src="https://user-images.githubusercontent.com/321279/157334438-071bac03-1a0d-4a80-960d-b6a0f70a8085.png">

**Comment by jorenbroekema** [comment]
Yeah in the playground it's correct, and it's inside an options object. I guess it was moved into the options object for some default formats but not others?

```js
{
  "files": [{
    "destination": "map.scss",
    "format": "scss/map-deep",
    "mapName": "my-tokens"
  }]
}
```
versus
```js
{
  "files": [{
    "destination": "_variables.css",
    "format": "css/variables",
    "options": {
      "selector": ":host"
    }
  }]
}
```
